Assets/Mirror/Runtime/NetworkWriter.cs

```diff
@@ -56,18 +50,22 @@ public void SetLength(long value)
             stream.SetLength(value);
         }
 
-        [Obsolete("Use WriteUInt16 instead")]
-        public void Write(ushort value) => WriteUInt16(value);
+        public void WriteByte(byte value) => stream.WriteByte(value);
+
+        // for byte arrays with consistent size, where the reader knows how many to read
+        // (like a packet opcode that's always the same)
+        public void WriteBytes(byte[] buffer, int offset, int count)
```

Comment:
what about moving this one into the extensions class too?
ultimately it would make sense to have all reads/writes in there. having a few write methods here and a few methods there would be a bit odd.

Reply:
This method isn't used by the weaver i think

Reply:
I can't. The methods left behind are the low level ones that touch stream (which is private). Extension methods can't touch private.

Basically, the organization is: NetworkWriter knows how to write basic stuff such as bytes and byte[]. Everything else is built on top of these and is in extensions